provision-team/service_check.sh

```diff
@@ -41,10 +41,12 @@ echo -e "DNS Url:" $dnsUrl"\n"
 poi_URL=$dnsUrl"/api/healthcheck/poi"
 user_URL=$dnsUrl"/api/healthcheck/user"
 trips_URL=$dnsUrl"/api/healthcheck/trips"
+user_profile_URL=$dnsUrl"/api/healthcheck/user-java"
```

Comment:
I believe you're looking for this URL
https://github.com/Azure-Samples/openhack-devops-team/blob/c3a7ba6a4b43b3a5522a69d84773d0ce87c82f45/apis/user-java/helm/values.yaml#L27